plugins/woocommerce-blocks/assets/js/blocks/product-template/style.scss

```diff
@@ -48,12 +48,18 @@ $min-product-width: 150px;
 		grid-gap: $grid-gap;
 
 		@for $i from 2 through 6 {
-			$gap-count: calc(#{ $i } - 1);
-			$total-gap-width: calc(#{ $gap-count } * #{ $grid-gap });
-			$max-product-width: calc((100% - #{ $total-gap-width }) / #{ $i });
+			$gap-count: calc(#{$i} - 1);
+			$total-gap-width: calc(#{$gap-count} * #{$grid-gap});
+			$max-product-width: calc((100% - #{$total-gap-width}) / #{$i});
 
 			&.columns-#{ $i } {
-				grid-template-columns: repeat(auto-fill, minmax(max(#{ $min-product-width }, #{ $max-product-width }), 1fr));
+				grid-template-columns: repeat(
+					auto-fill,
+					minmax(
+						max(#{$min-product-width}, #{$max-product-width}),
+						1fr
+					)
+				);
```

Comment:
Nitpick: I don't know if these changes are expected, it's just formatting though. I see the CI didn't run on this PR so let's see if those are passing linting. If not, they should be brought back to initial state.

Reply:
You are right. Stylelint wasn't working as expected on my VSCode. I spent some time fixing it, and now it works as expected. This will be helpful for me in the future as well 😅